WP_Auth0.php

```diff
@@ -441,11 +441,6 @@ public static function uninstall() {
 		delete_option( 'widget_wp_auth0_widget' );
 		delete_option( 'widget_wp_auth0_social_amplification_widget' );
 
-		delete_option( 'wp_auth0_client_grant_failed' );
```

Comment:
Options are removed upon plugin update so no need to remove when the plugin is uninstalled.